collections/_infographics/energy/electricity-per-capita-world.md

```diff
@@ -14,4 +14,29 @@ data-orig:
   - [ "UN (population)", "https://population.un.org/wpp/Download/Files/1_Indicators%20(Standard)/EXCEL_FILES/1_Population/WPP2019_POP_F01_1_TOTAL_POPULATION_BOTH_SEXES.xlsx" ]
 ---
 
-{% include includes-local/comment-placeholder.md slug="infografiky/elektrina-na-osobu-svet" %}
+## How to read this graph
+
+* The graph shows the **worldwide development of electricity produced per person by individual sources in the years 2000-2019**, as well as a comparison with selected world regions. Values are given in **<glossary id="w">kWh</glossary> of electricity produced per person per year**.
+* All graphs have the same vertical axis, so for regions with low average consumption per person, the filled part occupies only a minimal area of the graph.
+* However, the per capita comparison does not account for the different sizes of each region, so **the right column completes the regions' share of total global electricity generation in 2019**. The supplementary infographic, [Electricity generation in world regions](/infographics/electricity-per-capita-world), compares regions in terms of total electricity generation.
```

Comment:
"the different sizes of each region" to "the different sizes of the regions"

Reply:
Thanks. Updated as well